### PR TITLE
[Improvement] add name to export / set name of custom layout on import if na…

### LINKED ISSUE #5023: Add CustomLayout "name" to CustomLayout  JSON export
Hello,

We are using CustomLayout export / import and i wonder why the "name" field is not exported : 

https://github.com/pimcore/pimcore/blob/b9cb305e60518a3b4968d9227fb1f8a11603b43d/bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php#L631

It's would be easier instead of retrieving the name from the filename...

Thanks for considering this.

### PR DESCRIPTION
**Pull request by dpfahlbusch**
…me is available in custom layout json file

## Changes in this pull request  
Resolves #5023

### CODE REVIEW

**Review by kingjia90 — COMMENTED** [review]
Thank you for the PR!

It works, but there's one case: when you set the same name of an existing custom layout, the error thrown is unclear. 
image
Could you please add a check if the name is an existing or improve the error message a little bit mentioning the possible causes?

**Comment by aryaantony92** [comment]
@dpfahlbusch Just a gentle reminder. Could you please add a check if the name already exists or improve the error message a little bit mentioning the possible causes?

File: bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/customLayoutEditor.js [hunk 1]
```diff
@@ -997,7 +997,13 @@ pimcore.object.helpers.customLayoutEditor = Class.create({
 
             });
         }.bind(this), function () {
```
Comment (aryaantony92) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/customLayoutEditor.js]:
```suggestion
        }.bind(this), function (response) {
```

File: bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php [hunk 2]
```diff
@@ -706,6 +721,7 @@ public function exportCustomLayoutDefinitionAction(Request $request)
             if ($customLayout) {
                 $name = $customLayout->getName();
                 $customLayoutData = [
+                    'name' => $customLayout->getName(),
```
Comment (aryaantony92) [inline bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php]:
```suggestion
                    'name' => $name,
```

**Comment by aryaantony92** [comment]
@dpfahlbusch Thank you for the PR

**Pull request merged**